manifests/mcollective.pp

```diff
@@ -1,7 +1,7 @@
 define clamps::mcollective (
-  $user = $title,
-  $amqpass = 'password',
-  $amqserver = '$::server',
+  $user       = $title,
+  $amqpass    = 'password',
+  $amqservers = [$::servername],
```

Comment:
So servername is basically `facter fqdn` is that value going to match the CN of the certificate ( as you will be using address to connect to the host and thus the certificate it presents must have this address in it.